org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/component/WListbox.java

```diff
@@ -1277,6 +1289,10 @@ else if (c == Double.class)
 			{
 				setValueAt(total[col] , row -1 , col);
 			}
+			else if (c == Integer.class)
```

Comment:
Hi @nmicoud - as the three cases of the if execute the same code - this can be better written as:
```java
	if (c == BigDecimal.class || c == Double.class || c == Integer.class)
```

Reply:
Hi @CarlosRuiz-globalqss 
Done
(I didn't do that change initially as I wanted first to also reduce the way total[col] is calculated - but I didn't find a way to make it work - that's why I let the 3 if - anyway, is better this way).